armi/settings/fwSettings/globalSettings.py

```diff
@@ -227,6 +228,12 @@ def defineSettings() -> List[setting.Setting]:
             "of code that execute during a run",
             isEnvironment=True,
         ),
+        setting.Setting(
+            CONF_COVERAGE_CONFIG_FILE,
+            default="",  # should default be empty string, or the armi .coveragerc file?
```

Comment:
Commenting here so I remember to delete the comment. Also this comment on a comment is a question. What do you think, reviewer?

Reply:
I guess it depends on how the default is used.

I kind of like the idea of having the ARMI `.coveragerc` as the default. But.. I think we should leave it as the empty string.  Because the ARMI `.coveragerc` file path would be defined as that stupid global variable. And these settings should be independent of the current state of ARMI global context variables.

Reply:
Good point. I will clean this up now!